3.4/administration-cluster.md

```diff
@@ -208,3 +208,53 @@ To distribute shards onto the new _DBServer_ either click on the
 `Distribute Shards` button at the bottom of the `Shards` page in every
 database.
 
+This process can be monitored using the following javascript, that can give a 
+count of shards that still need to move. Such a script would therefore be a count 
+down to when the process finishes.
+
+```
+var dblist = db._databases();
+var internal = require("internal");
+var arango = internal.arango;
+var count = 0;
+
+var server = ARGUMENTS[0];
+
+for (dbase in dblist) {
+    var sd = arango.GET("/_db/" + dblist[dbase] + "/_admin/cluster/shardDistribution");
+    var collections = sd.results;
+    for (collection in collections) {
+        var current = collections[collection].Current;
+        for (shard in current) {
+            if (current[shard].leader == server) {
+                ++count;
+            }
+        }
+    }
+}
+console.log("Server %s count is %d", server, count)
+```
+
+This script has to be executed in the [`arangosh`](programs-arangosh.html) by issuing the following command:
+
+```
+arangosh  --server.username <username> --server.password <password> --javascript.execute <path-to-the-serverCleanMonitor.js> -- DBServer<number>
+```
+
+The output should be similar to the one below:
+
+```
+arangosh --server.username root --server.password pass --javascript.execute ~./serverCleanMonitor.js -- DBServer0002
+[7836] INFO Server DBServer0002 count is 1
+
+arangosh --server.username root --server.password pass --javascript.execute ~./serverCleanMonitor.js -- DBServer0002
+[7842] INFO Server DBServer0002 count is 0
+```
```

Comment:
The user is supposed to invoke arangosh from time to time and check if the count reached 0 already. Would it be beneficial if the script would instead poll the server every so often and sleep in between, until the count reaches zero, and print a status message every now and then so that you know it still does something?

Reply:
Script was update by this [commit](https://github.com/arangodb/docs/pull/30/commits/d6538448d09b55432447cca564a55d0db86710bc)